adyen.php

```diff
@@ -20,6 +20,8 @@
  * See the LICENSE file for more info.
  */
 
+// phpcs:disable PSR1.Classes.ClassDeclaration
+
```

Comment:
What does this do exactly ?

Reply:
Disables warning about a class not being part of a namespace.